src/test/java/BQJDBC/QueryResultTest/QueryResultTest.java

```diff
@@ -238,7 +238,7 @@ public void QueryResultTest03() {
             result = stmt.executeQuery(sql);
         } catch (SQLException e) {
             this.logger.debug("SQLexception" + e.toString());
-            Assert.assertTrue(e.toString().contains("Not found: Table measurement-lab:m_lab.2010_01"));
+                Assert.assertTrue(e.toString().contains("Not found: Table guid754187384106:m_lab.2010_01"));
```

Comment:
The build was failing here interestingly. How did it pass before?
Clearly the SQL above was "guid754187384106:m_lab.2010_01"

The Exception was just never being thrown ?